staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -4068,7 +4068,7 @@ type PodStatus struct {
 	ContainerStatuses []ContainerStatus `json:"containerStatuses,omitempty" protobuf:"bytes,8,rep,name=containerStatuses"`
 	// The Quality of Service (QOS) classification assigned to the pod based on resource requirements
 	// See PodQOSClass type for available QOS classes
-	// More info: https://git.k8s.io/community/contributors/design-proposals/node/resource-qos.md
```

Comment:
I think we should link to https://kubernetes.io/docs/concepts/workloads/pods/pod-qos/#quality-of-service-classes instead of an archived design doc